web/themes/custom/customtheme/templates/block/block--customtheme-page-title.html.twig

```diff
@@ -1,10 +1,12 @@
 <div {{ bem('block') }}>
   {{ title_prefix }}
   {% if label %}
-    <h2>Category</h2>
+    <h2 class="category_title">Category</h2>
```

Comment:
Use translatable rows for string. It can be trans block or |t.

{% trans %}
    Category
{% endtrans %}